meinberlin/config/settings/production.py

```diff
@@ -1,6 +1,14 @@
 from .base import *
 
 DEBUG = False
+
+CACHES = {
+    "default": {
+        "BACKEND": "django.core.cache.backends.redis.RedisCache",
```

Comment:
We'll also need to add this to `travis.py` otherwise it will use the `LocMemCache` from below